engines/agi/detection.cpp

```diff
@@ -53,21 +53,21 @@ static const PlainGameDescriptor agiGames[] = {
 	{"caitlyn", "Caitlyn's Destiny"},
 	{"ddp", "Donald Duck's Playground"},
 	{"goldrush", "Gold Rush!"},
-	{"kq1", "King's Quest I: Quest for the Crown"},
+	{"kq1", "King's Quest: Quest for the Crown"},
 	{"kq2", "King's Quest II: Romancing the Throne"},
 	{"kq3", "King's Quest III: To Heir Is Human"},
 	{"kq4", "King's Quest IV: The Perils of Rosella"},
 	{"lsl1", "Leisure Suit Larry in the Land of the Lounge Lizards"},
 	{"mickey", "Mickey\'s Space Adventure"},
 	{"mixedup", "Mixed-Up Mother Goose"},
-	{"mh1", "Manhunter 1: New York"},
+	{"mh1", "Manhunter: New York"},
 	{"mh2", "Manhunter 2: San Francisco"},
-	{"pq1", "Police Quest I: In Pursuit of the Death Angel"},
+	{"pq1", "Police Quest: In Pursuit of the Death Angel"},
 	{"serguei1", "Serguei's Destiny 1"},
 	{"serguei2", "Serguei's Destiny 2"},
 	{"sq0", "Space Quest 0: Replicated"},
-	{"sq1", "Space Quest I: The Sarien Encounter"},
-	{"sq2", "Space Quest II: Vohaul's Revenge"},
+	{"sq1", "Space Quest: Chapeter I - The Sarien Encounter"},
```

Comment:
Are you sure it is spelt like this on the box?

Reply:
For that one yes, "https://en.wikipedia.org/wiki/Space_Quest_I" and "https://en.wikipedia.org/wiki/Space_Quest_II" ,for the others I knew they had updates but I didn't connect that they could have been the remasters or the updated version on the separate engines: pretty obvious now TIL. So just so I am crystal clear for when I do much more complicated changes, also thank you for taking the time to respond, The notation should be a small commit with what systems are changed in the SYSTEM part and then described below that in a paragraph for pull requests(I am guessing the paragraph is only really needed for the pull request)? I did read the documentation but I also learn better from just doing it and messing up too so I appreciate this a lot.

Example of how I am thinking this should look in pull request:
SYSTEM: ARGOS, SCI, etc

Changed title of games X Y Z

and i Understand your example for commit thank you.

Reply:
Also is it important to include what issue number I am fixing?

Reply:
Typo: "Chapeter", should be "Chapter"